README.md

```diff
@@ -1,2 +1,17 @@
 # Reusable GitHub Actions workflows
 Maintained by Team Plattform.
+
+## Policy check skipped and why
+
+### Postgres module
```

Comment:
maybe remove this heading, or add for those below?

Reply:
this is borderline too nitpicky but it was not clear what was documented to me so, what do you think?